### PR TITLE
Cc fetch all data

### LINKED ISSUE #31: Replace fetch file data with the local data sets
* import file to scripts.js
* add global obj of data

### PR DESCRIPTION
**Pull request by carlymclift**
### What’s this PR do?
* fix method that was returning users average sleep to be 5.48 out of 5. Changed method to divide by length of this.sleepData. It's working now.
* Link fetch file to scripts.js

### Where should the reviewer start?
scripts.js

### How should this be manually tested?
open localhost:8080

resolves #31

### CODE REVIEW

File: src/Sleep.js [hunk 1]
```diff
@@ -35,7 +33,7 @@ class Sleep {
       sumSoFar += dataItem.sleepQuality;
       return sumSoFar;
     }, 0)
-    return totalSleepQuality / sleepData.length
+    return totalSleepQuality / this.sleepData.length
```
Comment (relyt4me) [inline src/Sleep.js:36]: I might have already shifted this in my refactor, good catch though

Comment (jordy1611) [reply src/Sleep.js:36]: I don't think I would have caught that! Nice!

File: src/scripts.js [hunk 2]
```diff
@@ -1,43 +1,54 @@
...
 let activityRepo;
 let currentUser;
 
-window.onload = startApp(); //Starts Here (global variables currentInformation = {})
+window.onload = startApp();
 
 function startApp() {
-  instantiatePageData();
-  populatePage();
+  fetchData()
```
Comment (jordy1611) [inline src/scripts.js:28]: Is this a function imported from the fetchAllData file?

File: src/scripts.js [hunk 3]
```diff
@@ -1,43 +1,54 @@
...
 
 function startApp() {
-  instantiatePageData();
-  populatePage();
+  fetchData() 
+    .then(allData => {
+      data.userData = allData.userData;
+      data.sleepData = allData.sleepData;
+      data.activityData = allData.activityData;
+      data.hydrationData = allData.hydrationData;
```
Comment (jordy1611) [inline src/scripts.js:33]: That fetchAllData function is returning the data sets so we have to 1) make is a synchronous process and 2) reassign are data files to whatever's been fetched?

File: src/scripts.js (hunk 2 above)
Comment (jordy1611) [reply src/scripts.js:28]: Yes it is!

**Pull request merged**